docs/api_reference/src/quantities.tex

```diff
@@ -2756,46 +2852,239 @@
+    requires @\exposconceptnc{InvocableQuantities}@<std::divides<>, quantity, quantity<R2, Rep2>>
+  friend constexpr @\libconcept{Quantity}@ auto operator/(const Q& lhs, const quantity<R2, Rep2>& rhs);
+
+  template<std::@\stdconcept{derived_from}@<quantity> Q, typename Value>
+    requires(!@\libconcept{Quantity}@<Value>) && (!@\libconcept{Reference}@<Value>) &&
+            @\exposconceptnc{InvokeResultOf}@<quantity_spec.character, std::multiplies<>, Rep, const Value&>
+  friend constexpr @\libconcept{QuantityOf}@<quantity_spec> auto operator*(const Q& lhs, const Value& rhs);
+  template<typename Value, std::@\stdconcept{derived_from}@<quantity> Q>
+    requires(!@\libconcept{Quantity}@<Value>) && (!@\libconcept{Reference}@<Value>) &&
+            @\exposconceptnc{InvokeResultOf}@<quantity_spec.character, std::multiplies<>, const Value&, Rep>
+  friend constexpr @\libconcept{QuantityOf}@<quantity_spec> auto operator*(const Value& lhs, const Q& rhs);
+  template<std::@\stdconcept{derived_from}@<quantity> Q, typename Value>
+    requires(!@\libconcept{Quantity}@<Value>) && (!@\libconcept{Reference}@<Value>) &&
+            @\exposconceptnc{InvokeResultOf}@<quantity_spec.character, std::divides<>, Rep, const Value&>
+  friend constexpr @\libconcept{QuantityOf}@<quantity_spec> auto operator/(const Q& lhs, const Value& rhs);
+  template<typename Value, std::@\stdconcept{derived_from}@<quantity> Q>
+    requires(!@\libconcept{Quantity}@<Value>) && (!@\libconcept{Reference}@<Value>) &&
+            @\exposconceptnc{InvokeResultOf}@<quantity_spec.character, std::divides<>, const Value&, Rep>
+  friend constexpr @\libconcept{QuantityOf}@<inverse(quantity_spec)> auto operator/(const Value&, const Q&);
+
+  // \ref{qty.cmp}, comparison
+
+  template<std::@\stdconcept{derived_from}@<quantity> Q, auto R2, typename Rep2>
+    requires @\seebelownc@
+  friend constexpr bool operator==(const Q& lhs, const quantity<R2, Rep2>& rhs);
+  template<std::@\stdconcept{derived_from}@<quantity> Q, auto R2, typename Rep2>
+    requires @\seebelownc@
+  friend constexpr bool operator<=>(const Q& lhs, const quantity<R2, Rep2>& rhs);
+
+  template<std::@\stdconcept{derived_from}@<quantity> Q, @\libconcept{RepresentationOf}@<quantity_character::scalar> Value>
+    requires @\seebelownc@
+  friend constexpr bool operator==(const Q& lhs, const Value& rhs);
+  template<std::@\stdconcept{derived_from}@<quantity> Q, @\libconcept{RepresentationOf}@<quantity_character::scalar> Value>
+    requires @\seebelownc@
+  friend constexpr bool operator<=>(const Q& lhs, const Value& rhs);
 };
 
+template<@\libconcept{Representation}@ Value, @\libconcept{Reference}@ R>
+explicit(false) quantity(Value v, R) -> quantity<R{}, Value>;
+
+template<@\libconcept{Representation}@ Value>
+explicit(false) quantity(Value) -> quantity<one, Value>;
+
+template<@\libconcept{QuantityLike}@ Q>
+quantity(Q)
+  -> quantity<quantity_like_traits<Q>::reference, typename quantity_like_traits<Q>::rep>;
```

Comment:
`explicit(...)` is missing

Reply:
And I think it should remain like that: <https://wg21.link/LWG3451>.
`explicit` in deduction guides don't do anything.

Reply:
Well, it does a bit: https://godbolt.org/z/GsqsY8qj4. It may additionally require direct initialization, even for an implicit constructor. But yes, it probably does not make sense to use it that way.

Reply:
Does it mean that we should drop the `explicit` specifiers from all CTADs?

Reply:
So it does do something.
Now I know: <https://wg21.link/over.match.class.deduct#5.sentence-3>.

> Does it mean that we should drop the `explicit` specifiers from all CTADs?

I haven't checked `quantity_point`'s.
`quantity`'s just repeats what the chosen constructor would do.
I prefer using deduction guides to only guide deduction.
That means leaving everything else to the chosen constructors.